CHANGELOG.adoc

```diff
@@ -115,6 +115,7 @@ The implemented fix does not allow to evaluate an AQL expression, but only to re
 - https://github.com/eclipse-sirius/sirius-web/issues/3206[#3206] [diagram] Fix an issue where precondition on tool was not respected during palette creation
 - https://github.com/eclipse-sirius/sirius-web/issues/3257[#3257] [diagram] Fix an issue that caused an infinite loop when selecting elements with shift.
 - https://github.com/eclipse-sirius/sirius-web/issues/3239[#3239] [form] Add missing reference widget view builder
+- https://github.com/eclipse-sirius/sirius-web/issues/3267[#3267] [tree] Fix an issue where `null` values were not filtered in `TreeComponent`.
```

Comment:
The goal of the representation cannot be to be prevent any potential mistakes that can occur. The contract is quite clear on the tree side "give us a list of elements to consider". So why are there null values in here?

Reply:
And if the issue is coming from the related elements view, that's where it should be fixed and if things are working nicely in the related elements view (for example, we ask for something in the EMF Edit part and we receive invalid values) then it should be fixed in the metamodel.

Reply:
Indeed, I have a derived reference in my metamodel who can return `null` values. I fixed it and everything seems ok now.